src/components/views/dialogs/CreateRoomDialog.js

```diff
@@ -176,7 +184,7 @@ export default createReactClass({
             <BaseDialog className="mx_CreateRoomDialog" onFinished={this.props.onFinished}
                 title={title}
             >
-                <form onSubmit={this.onOk}>
+                <form onSubmit={this.onOk} onKeyDown={this._onKeyDown}>
```

Comment:
shouldn't the create button just have `type=submit`?

Reply:
Hmm, that doesn't seem to work. Instead, the cancel button of `<DialogButtons>` is getting a click event on enter. Changing it in `<DialogButtons>` is also a bit of a wider change. Would rather keep the change small as it is currently if fine by you.